doc/classes/TextEdit.xml

```diff
@@ -265,6 +424,19 @@
 		<method name="get_total_gutter_width" qualifiers="const">
 			<return type="int" />
 			<description>
+				Get the total width of all gutters and internal padding.
+			</description>
+		</method>
+		<method name="get_total_visible_line_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Gets the total amount of lines that could be draw.
+			</description>
+		</method>
+		<method name="get_version" qualifiers="const">
+			<return type="int" />
+			<description>
+				Gets the current version of the [TextEdit].
```

Comment:
I think the "version" should be clarified, i.e. mention that it's related to undo history.